staging/src/k8s.io/apiserver/pkg/audit/policy/reader.go

```diff
@@ -73,10 +73,15 @@ func LoadPolicyFromBytes(policyDef []byte) (*auditinternal.Policy, error) {
 	}
 
 	// Ensure the policy file contained an apiVersion and kind.
-	if !apiGroupVersionSet[schema.GroupVersion{Group: gvk.Group, Version: gvk.Version}] {
+	gv := schema.GroupVersion{Group: gvk.Group, Version: gvk.Version}
+	if !apiGroupVersionSet[gv] {
 		return nil, fmt.Errorf("unknown group version field %v in policy", gvk)
 	}
 
+	if gv != auditv1.SchemeGroupVersion {
+		klog.Warningf("all resources under the alpha/beta version were deprecated - use %q instead", auditv1.SchemeGroupVersion)
```

Comment:
`%q is deprecated and will be removed in a future release, use %q instead` is better to show what's being used and how to replace it.

Reply:
@soltysh thanks for your review. 

updated